src/Microsoft.ML.TensorFlow/TensorFlow/Tensor.cs

```diff
@@ -409,6 +409,32 @@ public unsafe TFTensor(long value)
         /// <param name="data">Data.</param>
         public TFTensor(Complex[] data) : base(SetupTensor(TFDataType.Complex128, data, size: 16)) { }
 
+        public static unsafe TFTensor CreateString(byte[] buffer)
```

Comment:
>CreateString [](start = 38, length = 12)

It's nice to have summary comment on public methods. Or make it internal/protected :)